.gitignore

```diff
@@ -6,6 +6,8 @@ tags
 
 /core/src/main/resources/core/common-secrets.yml
 
+.graphqlconfig
```

Comment:
This is a configuration file for an IntelliJ plugin/node app. It contains the authorization token, which is why I'm forcibly ignoring it.